csharp/private/repositories.bzl

```diff
@@ -10,507 +10,11 @@ def csharp_repositories():
-        sha256 = "13f553e5815246d2b0d715944a8af9577784f8176ac6c5525e43b9eb48bf29fe",
-    )
-
-    nuget_package(
-        name = "System.Security.Cryptography.Primitives",
-        package = "System.Security.Cryptography.Primitives",
-        version = "4.3.0",
-        sha256 = "7e7162ec1dd29d58f96be05b8179db8e718dbd6ac2114e87a7fc23b235b3df5f",
-    )
-
-    nuget_package(
-        name = "System.Security.Cryptography.X509Certificates",
-        package = "System.Security.Cryptography.X509Certificates",
-        version = "4.3.2",
-        sha256 = "b24680b48aa291b06fd79f3a1287128b083e42a06cf6de6329402bfd06fdca2d",
-    )
-
-    nuget_package(
-        name = "System.Text.Encoding",
-        package = "System.Text.Encoding",
-        version = "4.3.0",
-        sha256 = "19cb475462d901afebaa404d86c0469ec89674acafe950ee6d8a4692e3a404b8",
-    )
-
-    nuget_package(
-        name = "System.Text.Encoding.Extensions",
-        package = "System.Text.Encoding.Extensions",
-        version = "4.3.0",
-        sha256 = "bee7c75e0f1000ac4796e8cf1c772bb46c00a859ac083e872a37c30221f20187",
-    )
-
-    nuget_package(
-        name = "System.Text.RegularExpressions",
-        package = "System.Text.RegularExpressions",
-        version = "4.3.1",
-        sha256 = "0f1b046749e73e8cf20b55b5eb8cab3145f09c07474a14bd127ec8983fc624c3",
-    )
-
-    nuget_package(
-        name = "System.Threading",
-        package = "System.Threading",
-        version = "4.3.0",
-        sha256 = "643437751e29cd5c266aa060e2169c65a55e9d0ff7c8017fb95ec15d95e38967",
-    )
-
-    nuget_package(
-        name = "System.Threading.Tasks",
-        package = "System.Threading.Tasks",
-        version = "4.3.0",
-        sha256 = "679ad77c9d445e9dc6df620a646899ea4a0c8d1bb49fc0b5346af0a5d21e9f8c",
-    )
-
-    nuget_package(
-        name = "System.Threading.Tasks.Extensions",
-        package = "System.Threading.Tasks.Extensions",
-        version = "4.5.3",
-        sha256 = "f138256c2e8a0479437927e0afa7797469fbc2ef2d7785c4465d89532a34f93c",
-    )
-
-    nuget_package(
-        name = "System.Threading.Timer",
-        package = "System.Threading.Timer",
-        version = "4.3.0",
-        sha256 = "a6686c96685084fdf64d66c1ce82132d9e01a0e441a98936e4e59baeed38a7db",
-    )
-
-    nuget_package(
-        name = "System.Xml.ReaderWriter",
-        package = "System.Xml.ReaderWriter",
-        version = "4.3.1",
-        sha256 = "7e32092b98e9cfdf6038dd933694eada3c37595b680852cef6b52f904fdbc995",
-    )
-
-    nuget_package(
-        name = "System.Xml.XDocument",
-        package = "System.Xml.XDocument",
-        version = "4.3.0",
-        sha256 = "ad6b5d72672e12534e4b309e85f9722b01e40d1a623a1249b3c09e4349750822",
+    native.new_local_repository(
+        name = "net",
+        path = "csharp/private/net",
+        build_file = "@d2l_rules_csharp//csharp/private:net.BUILD",
+        workspace_file = "@d2l_rules_csharp//csharp/private:net.WORKSPACE",
```

Comment:
I wonder if this file could just be `WORKSPACE` inside `csharp/private/net`? This is mostly a nit/I'm curious.

Reply:
Yeah probably, still playing

Reply:
Sorry I'll leave ya alone :P

Reply:
Nah it's all good